published_topic_names:  
  hazmat_alert : /vision/hazmat_alert
